.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Request a feature
+about: New ideas & improvements to Cesium are welcome.
+---
+
+<!-- 
+	Thanks for contributing to the Cesium Community! 
+
+	When suggesting an idea, give examples of the intended use case.
```

Comment:
Maybe also add that contributions are welcome - and even better than a feature request - and link to the related part of the Contributor's Guide.